src/Domain/Fund.php

```diff
@@ -183,4 +173,13 @@ public function addCampaignFunding(CampaignFunding $funding): void
 
         $this->campaignFundings->add($funding);
     }
+
+    public function allocationOrder(): int
+    {
+        return match ($this->fundType) {
+            'championFund' => 200,
+            'pledge' => 100,
+            'topupPledge' => 300,
+        };
+    }
```

Comment:
I think having all the order numbers together like this is quite valuable to be see at a glance what order they're used in (especially if we sorted them to numerical order in the code)